specification/loadtestservice/data-plane/Microsoft.LoadTestService/preview/2022-06-01-preview/loadtestservice.json

```diff
@@ -2185,8 +2185,8 @@
             "$ref": "#/definitions/TestModel"
           }
         },
-        "continuationToken": {
-          "description": "Continuation token for list of resources in case of paginated results, if applicable",
+        "nextLink": {
+          "description": "Token for the next list of resources in case of paginated results, if applicable",
```

Comment:
Update description this - Link for the next list of resources in case of paginated results, if applicable"